content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -390,73 +289,99 @@ run on. Each node has a maximum amount of local ephemeral storage it can provide
 
 The scheduler ensures that the sum of the resource requests of the scheduled Containers is less than the capacity of the node.
 
-### How Pods with ephemeral-storage limits run
+### Ephemeral storage consumption management {#resource-emphemeralstorage-consumption}
 
-For container-level isolation, if a Container's writable layer and logs usage exceeds its storage limit, the Pod will be evicted. For pod-level isolation, if the sum of the local ephemeral storage usage from all containers and also the Pod's emptyDir volumes exceeds the limit, the Pod will be evicted.
+If the kubelet is managing local ephemeral storage, then the kubelet
+measures storage use in:
 
-### Monitoring ephemeral-storage consumption
+- `emptyDir` volumes
+- log directories
+- writeable container layers
 
-When local ephemeral storage is used, it is monitored on an ongoing
-basis by the kubelet.  The monitoring is performed by scanning each
-emptyDir volume, log directories, and writable layers on a periodic
-basis.  Starting with Kubernetes 1.15, emptyDir volumes (but not log
-directories or writable layers) may, at the cluster operator's option,
-be managed by use of [project
-quotas](http://xfs.org/docs/xfsdocs-xml-dev/XFS_User_Guide/tmp/en-US/html/xfs-quotas.html).
-Project quotas were originally implemented in XFS, and have more
-recently been ported to ext4fs.  Project quotas can be used for both
-monitoring and enforcement; as of Kubernetes 1.16, they are available
-as alpha functionality for monitoring only.
+where these are backed by the system root filesystem.
 
-Quotas are faster and more accurate than directory scanning.  When a
-directory is assigned to a project, all files created under a
-directory are created in that project, and the kernel merely has to
-keep track of how many blocks are in use by files in that project.  If
-a file is created and deleted, but with an open file descriptor, it
-continues to consume space.  This space will be tracked by the quota,
-but will not be seen by a directory scan.
+If a Pod is using more ephemeral storage than you allow it to, the kubelet
+sets an eviction signal that triggers Pod eviction.
 
-Kubernetes uses project IDs starting from 1048576.  The IDs in use are
-registered in `/etc/projects` and `/etc/projid`.  If project IDs in
-this range are used for other purposes on the system, those project
-IDs must be registered in `/etc/projects` and `/etc/projid` to prevent
-Kubernetes from using them.
+For container-level isolation, if a Container's writable layer and log
+usage exceeds its storage limit, the kubelet marks the Pod for eviction.
+For pod-level isolation, if the sum of the local ephemeral storage usage
+from all containers and also the Pod's `emptyDir` volumes exceeds the limit,
```

Comment:
It might be helpful to clarify that the pod limit is the sum of the container limits.  I.E. Sum(Container usage) + Sum(emptydir usage) <= Sum(Container limit)